apps/gateway/Taskfile.yml

```diff
@@ -4,13 +4,24 @@ vars:
   app: gateway
 
 tasks:
+  generate-schema:
+    cmds:
+      - bash ./generate-schema.sh
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider making the generate-schema task platform-independent.

Using bash might limit the portability of this task. Consider using a more platform-independent approach or ensure bash is available in the execution environment.

```suggestion
  generate-schema:
    cmds:
      - ./generate-schema.sh
```